test/it/client-listUsers.js

```diff
@@ -1,11 +1,16 @@
-const assert = require('chai').assert;
-const faker = require('faker');
+const expect = require('chai').expect;
 const okta = require('../../');
 const models = require('../../src/models');
 const utils = require('../utils');
+const collection = require('../../src/collection');
+let orgUrl = process.env.OKTA_CLIENT_ORGURL;
```

Comment:
we should revisit this in the future. It'd be nice to use the same config parsing util that the client uses. If a customer wants to run the tests against an org that's setup via a config file, the env var solution won't work